packages/edit-site/src/components/page-pages/index.js

```diff
@@ -84,6 +73,9 @@ export default function PagePages() {
 			method: 'HEAD',
 			parse: false,
 		} ).then( ( res ) => {
+			// TODO: store this in core-data reducer and
```

Comment:
@youknowriad is there an open issue for this to follow?

Reply:
No, no issue at the moment, I was thinking about adding a todo list item to the data views overview issue

Reply:
This should be addressed in #55164